src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -98,7 +98,7 @@ export default defineComponent({
         selectedCourse,
         store.state.groupedRequirementFulfillmentReport,
         store.state.toggleableRequirementChoices,
-        store.state.overriddenFulfillmentChoices
+        /* deprecated AppOverriddenFulfillmentChoices */ {}
```

Comment:
`overriddenFulfillmentChoices` is all empty. Since it collides with the new name I choose, I decide just to replace it with `{}` to avoid confusion. The old `overriddenFulfillmentChoices` are not completely removed yet, to do so involves quite a lot of changes that will make this PR big and unfocused.